test/network_constructors.jl

```diff
@@ -18,6 +18,15 @@
 
     JuMP.@objective(ps_model.JuMPmodel, Min, AffExpr(0))
     JuMP.optimize!(ps_model.JuMPmodel)
+
+    if termination_status(ps_model.JuMPmodel) == MOI.OPTIMAL
```

Comment:
@daniel-thom Can you comment if this way of handling an error because a solution was not found is ok? Or should we add an exception for this error specifically?